reactions/src/SpecialFunctions.cpp

```diff
@@ -561,10 +556,10 @@ int funcn1_pseudo(double a, const double G[], double F[], void *params)
   b7 = beta1(k2,karg2,(k3*x1+k1*x2)/karg2);
 
 
-	mu1 = mu(a,k1,omega0,p1,p2,p3);
-	mu2 = mu(a,k2,omega0,p1,p2,p3);
-	mu3 = mu(a,karg1,omega0,p1,p2,p3);
-
+	mu1 = 1;//mu(a,k1,omega0,p1,p2,p3);
+	mu2 = 1;//mu(a,k2,omega0,p1,p2,p3);
+	mu3 = 1;//mu(a,karg1,omega0,p1,p2,p3);
+	mu4 = 1;//mu(a,karg4,omega0,p1,p2,p3);
```

Comment:
These are hardcoded to 1 rather than using mu because it's the pseudo case right? Is there ever a situation where you'd want to uncomment the mu() here?

Reply:
Yea you might want to use mu, for example in the unscreened approximation that Matteo used in his first reaction paper. I'll re arrange this and include a comment.